### ISSUE TITLE
Harpoon2 when pressing q/esc to exit UI, it moves to a marked item

### ISSUE THREAD

**Issue by whoop-t** [ThePrimeagen/harpoon#379]
Whichever item in the UI you have highlighted, then press q/esc(to exit ui), it will move to that item as if you pressed enter on it.
Neovim v0.9.0
Test on both Linux(popos) and Macos

Config:
```lua
  "ThePrimeagen/harpoon",
  lazy = false,
  branch = "harpoon2",
  config = function()
    local harpoon = require "harpoon"
    harpoon:setup()

    vim.keymap.set("n", "<leader>a", function() harpoon:list():append() end)
    vim.keymap.set("n", "<C-e>", function() harpoon.ui:toggle_quick_menu(harpoon:list()) end)
    vim.keymap.set("n", "<C-j>", function() harpoon:list():select(1) end)
    vim.keymap.set("n", "<C-k>", function() harpoon:list():select(2) end)
    vim.keymap.set("n", "<C-l>", function() harpoon:list():select(3) end)
    vim.keymap.set("n", "<C-;>", function() harpoon:list():select(4) end)
  end,
  dependencies = { "nvim-lua/plenary.nvim" },

```

**Comment by theKnightsOfRohan** [comment]
I've also been getting this error.

**Comment by whoop-t** [comment]
Confirming this is fixed with #392 
tyty, Closing
